schemas/questions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "question",
+      "description",
+      "type"
+    ],
+    "properties": {
+      "question": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "validation": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "FREESINGLE",
+          "FREEMULTIPLE",
+          "CHOICESINGLE",
+          "CHOICEMULTIPLE"
+        ]
+      },
+      "options": {
+        "oneOf": [
```

Comment:
why is the oneof here?

Reply:
Good point, this probably can be simplified to a simple list.

Reply:
The json schema has been simplified for options (and groups have been added as well).